databricks/koalas/plot/core.py

```diff
@@ -337,6 +338,43 @@ def get_fliers(colname, outliers, min_val):
         return fliers
 
 
+class KdePlotBase:
+    @staticmethod
+    def get_ind(sdf, ind):
+        # 'data' is a Spark DataFrame that selects one column.
```

Comment:
nit: `# 'sdf' is a ...`?

Reply:
Ah, I don't know why I keep missing these stuff. Thanks @ueshin.